evm/src/memory/segments.rs

```diff
@@ -52,9 +52,7 @@ pub(crate) enum Segment {
     TxnBloom = 26,
     /// Contains the computed bloom filter of a block.
     BlockBloom = 27,
```

Comment:
I think this segment can be removed now?

Reply:
Ah indeed, good catch